### PR TITLE
#2289 - Implement logic for Application Event Code in IER12 - moved application event code logic to services (Part 2) 

### PR DESCRIPTION
**Pull request by ann-aot**
- As mentioned, moved the application event code logic to `utils-service`.
- Reference: https://github.com/bcgov/SIMS/pull/2374#discussion_r1353664756
- There are no code change

**NOTE**
More PRs to come

### CODE REVIEW

**Review by guru-aot — APPROVED** [review]
LGTM, nice work Ann

**Review by andrepestana-aot — APPROVED** [review]
Looks good.

File: sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/utils-service/application-event-code-utils.service.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+          currentDisbursementSchedule,
+          activeRestrictionsActionTypes,
+        );
+      case ApplicationStatus.Cancelled:
+        return ApplicationEventCode.DISC;
+    }
+  }
+
+  @InjectLogger()
+  logger: LoggerService;
```
Comment (andrewsignori-aot) [inline sources/packages/backend/libs/integrations/src/institution-integration/ier12-integration/utils-service/application-event-code-utils.service.ts:58]: Usually we add the logger once we need it.

**Review by andrewsignori-aot — APPROVED** [review]
Thanks for doing the refactor, looks good 👍

**Pull request merged**